From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: SuperKali <hello@superkali.me>
Date: Sat, 29 Aug 2026 12:41:52 +0200
Subject: arm64: dts: qcom: cq8725s-ayn: point the codec at the USBSS switch

The integrated 3.5mm jack on these boards is wired through the
WCD9395 USBSS analog switch, and the codec driver looks the switch
up via a qcom,usbss phandle to route it into audio-accessory mode
when MBHC detects a mechanical insert. The phandle was never added,
so the switch stayed in safe mode, the HPH lines floated and the
jack never produced usable audio.

Signed-off-by: SuperKali <hello@superkali.me>
---
 arch/arm64/boot/dts/qcom/cq8725s-ayn-common.dtsi | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/qcom/cq8725s-ayn-common.dtsi b/arch/arm64/boot/dts/qcom/cq8725s-ayn-common.dtsi
index 111111111111..222222222222 100644
--- a/arch/arm64/boot/dts/qcom/cq8725s-ayn-common.dtsi
+++ b/arch/arm64/boot/dts/qcom/cq8725s-ayn-common.dtsi
@@ -40,6 +40,7 @@ wcd939x: audio-codec {
 		qcom,mbhc-headphone-vthreshold-microvolt = <50000>;
 		qcom,rx-device = <&wcd_rx>;
 		qcom,tx-device = <&wcd_tx>;
+		qcom,usbss = <&wcd_usbss>;
 
 		reset-gpios = <&tlmm 101 GPIO_ACTIVE_LOW>;
 
-- 
Armbian

